docs/examples/Configuring_QCoDeS.ipynb

```diff
@@ -4,7 +4,23 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Logging hadn't been started.\n",
+      "Activating auto-logging. Current session state plus future input saved.\n",
+      "Filename       : C:\\Users\\a-davidpoole\\.qcodes\\logs\\command_history.log\n",
+      "Mode           : append\n",
+      "Output logging : True\n",
+      "Raw input log  : False\n",
+      "Timestamping   : True\n",
+      "State          : active\n",
+      "Qcodes Logfile : C:\\Users\\a-davidpoole\\.qcodes\\logs\\211129-9908-qcodes.log\n"
+     ]
+    }
+   ],
    "source": [
     "%load_ext autoreload\n",
     "%autoreload 2\n",
```

Comment:
move this below the title

Reply:
@astafan8 I would remove this auto reload. IMHO auto reload is flaky at the best of times and creates a lot of issues

Reply:
totally agree!